tests/e2e/standard/contacts/contact-summary.wdio-spec.js

```diff
@@ -249,132 +246,120 @@ describe('Contact summary info', () => {
+    expect(await contactsPo.cardFieldText('uhc_stats_last_visited_date')).to.equal('');
 
-    expect(await contactsPo.cardFieldLabelText('uhc_stats_interval_start')).toBe('uhc_stats_interval_start');
+    expect(await contactsPo.cardFieldLabelText('uhc_stats_interval_start')).to.equal('uhc_stats_interval_start');
     const startDate = moment().startOf('month').valueOf().toString();
-    expect(await contactsPo.cardFieldText('uhc_stats_interval_start')).toBe(startDate);
+    expect(await contactsPo.cardFieldText('uhc_stats_interval_start')).to.equal(startDate);
 
-    expect(await contactsPo.cardFieldLabelText('uhc_stats_interval_end')).toBe('uhc_stats_interval_end');
+    expect(await contactsPo.cardFieldLabelText('uhc_stats_interval_end')).to.equal('uhc_stats_interval_end');
     const endDate = moment().endOf('month').valueOf().toString();
-    expect(await contactsPo.cardFieldText('uhc_stats_interval_end')).toBe(endDate);
+    expect(await contactsPo.cardFieldText('uhc_stats_interval_end')).to.equal(endDate);
 
-    expect(await contactsPo.cardFieldLabelText('alicetag')).toBe('aliceTag');
-    expect(await contactsPo.cardFieldText('alicetag')).toBe(`${ALICE.name} ${ALICE.phone}`);
+    expect(await contactsPo.cardFieldLabelText('alicetag')).to.equal('aliceTag');
+    expect(await contactsPo.cardFieldText('alicetag')).to.equal(`${ALICE.name} ${ALICE.phone}`);
 
-    expect(await contactsPo.cardFieldLabelText('davidtag')).toBe('davidTag');
-    expect(await contactsPo.cardFieldText('davidtag')).toBe(`${DAVID.name} ${DAVID.phone}`);
+    expect(await contactsPo.cardFieldLabelText('davidtag')).to.equal('davidTag');
+    expect(await contactsPo.cardFieldText('davidtag')).to.equal(`${DAVID.name} ${DAVID.phone}`);
 
-    expect(await contactsPo.cardFieldLabelText('visittag')).toBe('visitTag');
-    expect(await contactsPo.cardFieldText('visittag')).toBe(DAVID_VISIT.form);
+    expect(await contactsPo.cardFieldLabelText('visittag')).to.equal('visitTag');
+    expect(await contactsPo.cardFieldText('visittag')).to.equal(DAVID_VISIT.form);
 
     // assert that the pregnancy card exists and has the right fields.
-    expect(
-      await helper.getTextFromElementNative(
-        element(by.css('.content-pane .meta > div > .card .action-header h3'))
-      )
-    ).toBe('test.pregnancy');
-    expect(
-      await helper.getTextFromElementNative(
-        element(by.css('.content-pane .meta > div > .card .row label'))
-      )
-    ).toBe('test.visits');
-    expect(
-      await helper.getTextFromElementNative(
-        element(by.css('.content-pane .meta > div > .card .row p'))
-      )
-    ).toBe('1');
+    expect(await (await $('.content-pane .meta > div > .card .action-header h3')).getText()).to.equal('test.pregnancy');
+    expect(await ( await $('.content-pane .meta > div > .card .row label')).getText()).to.equal('test.visits');
+    expect(await (await $(('.content-pane .meta > div > .card .row p'))).getText() ).to.equal('1');
```

Comment:
Please create the corresponding selector variables for these three fields in the correct page-object file.